[2017-02-27 15:47:05] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /lucky/number"" at D:\xampp\htdocs\R_&_D\symfony2\app\cache\prod\classes.php line 2069 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /lucky/number\" at D:\\xampp\\htdocs\\R_&_D\\symfony2\\app\\cache\\prod\\classes.php:2069, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at D:\\xampp\\htdocs\\R_&_D\\symfony2\\app\\cache\\prod\\appProdProjectContainerUrlMatcher.php:39)"} []
[2017-02-27 15:50:02] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /main/number"" at D:\xampp\htdocs\R_&_D\symfony2\app\cache\prod\classes.php line 2069 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /main/number\" at D:\\xampp\\htdocs\\R_&_D\\symfony2\\app\\cache\\prod\\classes.php:2069, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at D:\\xampp\\htdocs\\R_&_D\\symfony2\\app\\cache\\prod\\appProdProjectContainerUrlMatcher.php:39)"} []
[2017-02-27 15:50:12] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /main/number"" at D:\xampp\htdocs\R_&_D\symfony2\app\cache\prod\classes.php line 2069 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /main/number\" at D:\\xampp\\htdocs\\R_&_D\\symfony2\\app\\cache\\prod\\classes.php:2069, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at D:\\xampp\\htdocs\\R_&_D\\symfony2\\app\\cache\\prod\\appProdProjectContainerUrlMatcher.php:39)"} []
[2017-02-27 15:56:35] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /main/number"" at D:\xampp\htdocs\R_&_D\symfony2\app\cache\prod\classes.php line 2069 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /main/number\" at D:\\xampp\\htdocs\\R_&_D\\symfony2\\app\\cache\\prod\\classes.php:2069, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at D:\\xampp\\htdocs\\R_&_D\\symfony2\\app\\cache\\prod\\appProdProjectContainerUrlMatcher.php:39)"} []
[2017-02-27 15:56:41] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /main/number"" at D:\xampp\htdocs\R_&_D\symfony2\app\cache\prod\classes.php line 2069 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /main/number\" at D:\\xampp\\htdocs\\R_&_D\\symfony2\\app\\cache\\prod\\classes.php:2069, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at D:\\xampp\\htdocs\\R_&_D\\symfony2\\app\\cache\\prod\\appProdProjectContainerUrlMatcher.php:39)"} []
[2017-02-28 09:08:32] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET /main/number"" at D:\xampp\htdocs\R_&_D\symfony2\app\cache\prod\classes.php line 2069 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /main/number\" at D:\\xampp\\htdocs\\R_&_D\\symfony2\\app\\cache\\prod\\classes.php:2069, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at D:\\xampp\\htdocs\\R_&_D\\symfony2\\app\\cache\\prod\\appProdProjectContainerUrlMatcher.php:39)"} []
